docs/dev/ci-reference.adoc

```diff
@@ -1,6 +1,6 @@
-= ci-reference
+= CI-reference
 
-== Running integration test job in Travis CI
+== Running integration test job on Travis CI
```

Comment:
AFAIK we are not running any test on travis CI in odo repo. Shouldn't we remove whole content about travis?

Reply:
Understood I will remove the Travis related information

Reply:
I removed the TravisCI releated docs from the Ci-reference.